DetectorDescription/Core/src/DDCompactView.cc

```diff
@@ -112,28 +109,29 @@ void DDCompactView::swap( DDCompactView& repToSwap ) {
 }
 
 DDCompactView::DDCompactView()
-  : rep_(new DDCompactViewImpl),
+  : rep_( new DDCompactViewImpl ),
```

Comment:
@kpedro88 - I'm not sure I prefer the verbose version:
```
: rep_( std::make_unique<DDCompactViewImpl>() ),
```
IMHO, using ```new``` is perfectly fine here
@Dr15Jones - please, comment

Reply:
It makes it clearer that smart pointers are being used. A bare `new` looks like a potential memory leak, even though it isn't in this case.

Reply:
I also prefer the use of `std::make_unique` for the same reasons as Kevin.

Reply:
Fixed in https://github.com/cms-sw/cmssw/pull/24378/commits/66ed9285c07525a08e43a0ee2c3a640d72ea95ac